---
title: "01_GenArch_Figs_2015-12-17"
author: "Heather E. Wheeler"
date: "`r Sys.time()`"
output: html_document
---

```{r, echo=TRUE, message=FALSE, warning=FALSE}
  library(ggplot2)
  library(reshape2)
  library(dplyr)
  library(tidyr)
  library(GGally)
  library(grid)
  library(corrplot)
  "%&%" = function(a,b) paste(a,b,sep="")
  source('/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/GenArchPaper/multiplot.R')
  my.dir <- '/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/'
  fig.dir <- '~/GitHub/GenArch/GenArchPaper/Figures/'
  my.vol <- '/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/BSLMM_exp/'
  rna.dir <- my.dir %&% "gtex-rnaseq/"
  out.dir <- rna.dir %&% "ind-tissues-RPKM/"
```
##Fig1
DGN-WB joint heritability. Local h^2^ is estimated with SNPs within 1 Mb of each gene. distal h^2^ is estimated with either all non-chr SNPs or SNPs that are eQTLs in the Framingham Heart Study on other chromosomes (FDR < 0.05).
```{r jointH2,fig.width=6,fig.height=16, echo=TRUE, warning=FALSE, message=FALSE,eval=TRUE}
otherfile<-my.dir %&% 'expArch_DGN-WB_imputedGTs/DGN-WB.h2.all.models_FHSfdr0.05.all.Chr1-22_globalOtherChr.2015-03-18.txt'

fdrother<-read.table(otherfile,header=T) ##FHS eQTLs w/fdr<0.05 on non-gene chromosomes used to define global GRM
d  <- fdrother %>% mutate(ymin = pmax(0, glo.jt.h2 - 2 * glo.jt.se), ymax = pmin(1, glo.jt.h2 + 2 * glo.jt.se) )
cigt0 <- d$ymin>0
ptrue<-round(table(cigt0)/sum(table(cigt0)),3)*100
ptrue
fdrother <- mutate(d, loc.jt.P = 2*pnorm(-abs(loc.jt.h2/loc.jt.se)), glo.jt.P = 2*pnorm(-abs(glo.jt.h2/glo.jt.se))) %>% mutate(loc.jt.P=ifelse(is.na(loc.jt.P),1,loc.jt.P),glo.jt.P=ifelse(is.na(glo.jt.P),1,glo.jt.P)) %>% mutate(locPlt05=loc.jt.P < 0.05,gloPlt05=glo.jt.P < 0.05) %>% mutate(`distal P`=factor(gloPlt05,labels=c('\u2265 0.05','< 0.05')))
table(fdrother$gloPlt05)

##Plot FDR based results
a<-ggplot(fdrother,aes(x=loc.jt.h2,y=glo.jt.h2,color=`distal P`)) + geom_point(cex=0.8) + geom_abline(intercept=1, slope=-1) + xlab(expression("local h"^2)) + ylab(expression("distal h"^2)) + coord_cartesian(ylim=c(-0.05,1.05),xlim=c(-0.05,1.05)) + theme_bw() +theme(axis.text=element_text(size=16),axis.title=element_text(size=18,face="bold"),legend.justification=c(1,1),legend.position=c(1,1))

##plot joint h2 estimates
local <- fdrother %>% select(loc.jt.h2,loc.jt.se,loc.jt.P,locPlt05)%>% arrange(loc.jt.h2) %>% mutate(loc.jt.h2=ifelse(is.na(loc.jt.h2),0,loc.jt.h2), loc.jt.se=ifelse(is.na(loc.jt.se),base::sample(loc.jt.se[is.na(loc.jt.se)==FALSE][1:100],size=length(loc.jt.se[is.na(loc.jt.se)==TRUE]),replace=TRUE),loc.jt.se))%>% arrange(loc.jt.h2) 
names(local) = c('h2','se','jt.P','Plt05')
data <- local %>% mutate(ymin = pmax(0, h2 - 2 * se), ymax = pmin(1, h2 + 2 * se) )
cigt0 <- data$ymin>0
cat("local jt.P < 0.05 (distal: known eQTLs)")
table(data$jt.P < 0.05,useNA='i')
sum(table(data$jt.P < 0.05,useNA='i'))
ptrue <-round(table(data$jt.P < 0.05,useNA='i')/sum(table(data$jt.P < 0.05,useNA='i')),3)*100
ptrue
meanh2<-round(mean(data$h2),3)
meanh2
meanse <- round(mean(data$se),3)
meanse
data <- mutate(data,P=factor(Plt05,labels=c("\u2265 0.05","< 0.05")),position=1:nrow(data))
my_grob2 = grobTree(textGrob(substitute(paste("% P < 0.05: ", m),list(m=ptrue[2])), x=0.05,  y=0.66, hjust=0,gp=gpar(fontsize=14)))

b<-ggplot(data,aes(x=position,y=h2,ymin=ymin, ymax=ymax, color=P) ) + geom_pointrange(col='gray')+geom_point()+ylab(expression("local h"^2)) + xlab(expression("genes ordered by local h"^2))+coord_cartesian(ylim=c(-0.05,1.05))+theme_bw()+theme(axis.text=element_text(size=16),axis.title=element_text(size=18,face="bold"),legend.justification=c(0,1),legend.position=c(0,1))+annotation_custom(my_grob2)

global <- fdrother %>% select(gene,glo.jt.h2,glo.jt.se,glo.jt.P,gloPlt05) %>% arrange(glo.jt.h2) %>% mutate(glo.jt.h2=ifelse(is.na(glo.jt.h2),0,glo.jt.h2), glo.jt.se=ifelse(is.na(glo.jt.se),base::sample(glo.jt.se[is.na(glo.jt.se)==FALSE][1:1000],size=length(glo.jt.se[is.na(glo.jt.se)==TRUE]),replace=TRUE),glo.jt.se))%>% arrange(glo.jt.h2)  
names(global) = c('gene','h2','se','jt.P','Plt05')
data <- global %>% mutate(ymin = pmax(0, h2 - 2 * se), ymax = pmin(1, h2 + 2 * se) )
#data <- data[complete.cases(data),]
cigt0 <- data$ymin>0
ptrueCI<-round(table(cigt0)/sum(table(cigt0)),3)*100
cat("global jt.P < 0.05 (distal: known eQTLs)")
table(data$jt.P < 0.05,useNA='i')
sum(table(data$jt.P < 0.05,useNA='i'))
ptrue <-signif(table(data$jt.P < 0.05,useNA='i')/sum(table(data$jt.P < 0.05,useNA='i')),3)*100
ptrue
meanh2<-round(mean(data$h2),3)
meanh2
meanse <- round(mean(data$se),3)
meanse
data <- mutate(data,P=factor(Plt05,labels=c("\u2265 0.05","< 0.05")),position=1:nrow(data))

glopriorlist <- dplyr::filter(data,Plt05==TRUE) %>% dplyr::select(gene)

my_grob2 = grobTree(textGrob(substitute(paste("% P < 0.05: ", m),list(m=ptrue[2])), x=0.05,  y=0.96, hjust=0,gp=gpar(fontsize=14)))

c<-ggplot(data,aes(x=position,y=h2,ymin=ymin, ymax=ymax, color=P) ) + geom_pointrange(col='gray')+geom_point()+ylab(expression("distal h"^2)) + xlab(expression("genes ordered by distal h"^2))+coord_cartesian(ylim=c(-0.05,1.05))+theme_bw()+theme(axis.text=element_text(size=16),axis.title=element_text(size=18,face="bold"),legend.position="none")+annotation_custom(my_grob2)

##make same plots using all SNPs on other chrs for global
otherfile<-my.dir %&% 'expArch_DGN-WB_imputedGTs/DGN-WB.h2.all.models_all.Chr1-22_globalOtherChrallSNPs.2015-10-22.txt'

other<-read.table(otherfile,header=T) ##all SNPs on non-gene chromosomes used to define global GRM
d  <- other %>% mutate(ymin = pmax(0, glo.jt.h2 - 2 * glo.jt.se), ymax = pmin(1, glo.jt.h2 + 2 * glo.jt.se) )
cigt0 <- d$ymin>0
ptrue<-round(table(cigt0)/sum(table(cigt0)),3)*100
ptrue

other <- mutate(d, loc.jt.P = 2*pnorm(-abs(loc.jt.h2/loc.jt.se)), glo.jt.P = 2*pnorm(-abs(glo.jt.h2/glo.jt.se))) %>% mutate(loc.jt.P=ifelse(is.na(loc.jt.P),1,loc.jt.P),glo.jt.P=ifelse(is.na(glo.jt.P),1,glo.jt.P)) %>% mutate(locPlt05=loc.jt.P < 0.05,gloPlt05=glo.jt.P < 0.05) %>% mutate(`distal P`=factor(gloPlt05,labels=c('\u2265 0.05','< 0.05')))
table(other$gloPlt05)

aother<-ggplot(other,aes(x=loc.jt.h2,y=glo.jt.h2,color=`distal P`)) + geom_point(cex=0.8) + geom_abline(intercept=1, slope=-1) + xlab(expression("local h"^2)) + ylab(expression("distal h"^2)) + coord_cartesian(ylim=c(-0.05,1.05),xlim=c(-0.05,1.05)) + theme_bw() + theme(axis.text=element_text(size=16),axis.title=element_text(size=18,face="bold"),legend.justification=c(1,1),legend.position=c(1,1))

local <- other %>% select(loc.jt.h2,loc.jt.se,loc.jt.P,locPlt05) %>% arrange(loc.jt.h2) %>% mutate(loc.jt.h2=ifelse(is.na(loc.jt.h2),0,loc.jt.h2), loc.jt.se=ifelse(is.na(loc.jt.se),base::sample(loc.jt.se[is.na(loc.jt.se)==FALSE][1:100],size=length(loc.jt.se[is.na(loc.jt.se)==TRUE]),replace=TRUE),loc.jt.se))%>% arrange(loc.jt.h2)  
names(local) = c('h2','se','jt.P','Plt05')
data <- local %>% mutate(ymin = pmax(0, h2 - 2 * se), ymax = pmin(1, h2 + 2 * se) )
cigt0 <- data$ymin>0
ptrueCI<-round(table(cigt0)/sum(table(cigt0)),3)*100
cat("local jt.P < 0.05 (distal: all eQTLs)")
table(data$jt.P < 0.05,useNA='i')
sum(table(data$jt.P < 0.05,useNA='i'))
ptrue <-round(table(data$jt.P < 0.05,useNA='i')/sum(table(data$jt.P < 0.05,useNA='i')),3)*100
ptrue
meanh2<-round(mean(data$h2),3)
meanh2
meanse <- round(mean(data$se),3)
meanse
data <- mutate(data,P=factor(Plt05,labels=c("\u2265 0.05","< 0.05")),position=1:nrow(data))
my_grob2 = grobTree(textGrob(substitute(paste("% P < 0.05: ", m),list(m=ptrue[2])), x=0.05,  y=0.66, hjust=0,gp=gpar(fontsize=14)))

bother<-ggplot(data,aes(x=position,y=h2,ymin=ymin, ymax=ymax, color=P) ) + geom_pointrange(col='gray')+geom_point()+ylab(expression("local h"^2)) + xlab(expression("genes ordered by local h"^2))+coord_cartesian(ylim=c(-0.05,1.05))+theme_bw()+theme(axis.text=element_text(size=16),axis.title=element_text(size=18,face="bold"),legend.justification=c(0,1),legend.position=c(0,1))+annotation_custom(my_grob2)

global <- other %>% select(gene,glo.jt.h2,glo.jt.se,glo.jt.P,gloPlt05) %>% arrange(glo.jt.h2) %>% mutate(glo.jt.h2=ifelse(is.na(glo.jt.h2),0,glo.jt.h2), glo.jt.se=ifelse(is.na(glo.jt.se),base::sample(glo.jt.se[is.na(glo.jt.se)==FALSE][1:1000],size=length(glo.jt.se[is.na(glo.jt.se)==TRUE]),replace=TRUE),glo.jt.se))%>% arrange(glo.jt.h2) 
names(global) = c('gene','h2','se','jt.P','Plt05')
data <- global %>% mutate(ymin = pmax(0, h2 - 2 * se), ymax = pmin(1, h2 + 2 * se) )
cigt0 <- data$ymin>0
ptrueCI<-round(table(cigt0)/sum(table(cigt0)),3)*100
cat("global jt.P < 0.05 (distal: known eQTLs)")
table(data$jt.P < 0.05,useNA='i')
sum(table(data$jt.P < 0.05,useNA='i'))
ptrue <-signif(table(data$jt.P < 0.05,useNA='i')/sum(table(data$jt.P < 0.05,useNA='i')),3)*100
ptrue
meanh2<-round(mean(data$h2,useNA="T"),3)
meanh2
meanse <- round(mean(data$se),3)
meanse
data <- mutate(data,P=factor(Plt05,labels=c("\u2265 0.05","< 0.05")),position=1:nrow(data))

glolist <- dplyr::filter(data,Plt05==TRUE) %>% dplyr::select(gene)
table(glolist$gene %in% glopriorlist$gene)

my_grob2 = grobTree(textGrob(substitute(paste("% P < 0.05: ", m),list(m=ptrue[2])), x=0.05,  y=0.96, hjust=0,gp=gpar(fontsize=14)))

cother<-ggplot(data,aes(x=position,y=h2,ymin=ymin, ymax=ymax, color=P) ) + geom_pointrange(col='gray')+geom_point()+ylab(expression("distal h"^2)) + xlab(expression("genes ordered by distal h"^2))+coord_cartesian(ylim=c(-0.05,1.05))+theme_bw()+theme(axis.text=element_text(size=16),axis.title=element_text(size=18,face="bold"),legend.position="none")+annotation_custom(my_grob2)

png(filename=fig.dir %&% "Fig-DGN-jt-h2.png",width=720,height=960)
multiplot(aother+ggtitle('local = SNPs within 1Mb of gene\ndistal = SNPs on non-gene chrs\n') + theme(plot.title=element_text(face="bold")), bother, cother, a+ggtitle('local = SNPs within 1Mb of gene\ndistal = known eQTLs on non-gene chrs\n') + theme(plot.title=element_text(face="bold")),b, c,cols=2)
dev.off()

tiff(filename=fig.dir %&% "Fig-DGN-jt-h2.tiff",width=720,height=960)
multiplot(aother+ggtitle('local = SNPs within 1Mb of gene\ndistal = SNPs on non-gene chrs\n') + theme(plot.title=element_text(face="bold")), bother, cother, a+ggtitle('local = SNPs within 1Mb of gene\ndistal = known eQTLs on non-gene chrs\n') + theme(plot.title=element_text(face="bold")),b, c,cols=2)
dev.off()
```

##Fig 2
###DGN Polygenic v. sparse  by elastic net.
```{r EN, echo=TRUE, warning=FALSE, message=FALSE,eval=TRUE}
data<-read.table(my.dir %&% 'DGN-WB_exp_10-foldCV_1-reps_elasticNet_eachAlphaR2_hapmap2snps_chr22_2015-01-21.txt',header=T)
colnames(data)<-c("gene",0:20/20)
a <- data$gene %in% fdrother$gene
table(a)
data <- data[a,]
ngenes<-dim(data)[1]
print("Elastic Net DGN-WB chr22 (" %&% ngenes %&% " genes)")
data_long<-melt(data,by=gene)
## # genes w/alpha 1 R2 > 0.3
table(data[,22] > 0.3)[2]
## Using gene as id variables
a <- ggplot(data_long, aes(x = as.numeric(levels(variable))[variable] , y = value), group=gene) + geom_line(lwd=0.5,show_guide = FALSE,linetype=1) + aes(color = gene) + xlab(expression(paste("elastic net mixing parameter (",alpha, ")"))) + ylab(expression(paste("10-fold cross-validation R"^2))) + theme_bw(base_size = 20) + coord_cartesian(ylim=c(0.3,1),xlim=c(-0.02,1.02))+ geom_point(show_guide = FALSE)
print(a)


###add all genes for Fig 3B blandaltman plot rather than just chr22
dgn.dir <- '/Volumes/im-lab/nas40t2/hwheeler/PrediXcan_CV/GTEx_2014-06013_release/transfers/PrediXmod/DGN-WB/DGN-calc-weights/'
alpha1 <- read.table(dgn.dir %&% 'DGN-WB_exp_10-foldCV_elasticNet_alpha1_hapmapSnpsCEU_chr1-22_2015-02-02.txt',header=TRUE) %>%  mutate(`1`=R2) %>% select(gene,`1`)
ngenesall <- length(unique(alpha1$gene))
ngenesall
alpha95 <- read.table(dgn.dir %&% 'DGN-WB_exp_10-foldCV_elasticNet_alpha0.95_hapmapSnpsCEU_chr1-22_2015-08-21.txt',header=TRUE) %>% mutate(`0.95`=R2) %>% select(gene,`0.95`)
alpha50 <- read.table(dgn.dir %&% 'DGN-WB_exp_10-foldCV_elasticNet_alpha0.5_hapmapSnpsCEU_chr1-22_2015-02-02.txt',header=TRUE) %>% mutate(`0.50`=R2) %>% select(gene,`0.50`)
alpha05 <- read.table(dgn.dir %&% 'DGN-WB_exp_10-foldCV_elasticNet_alpha0.05_hapmapSnpsCEU_chr1-22_2015-08-21.txt',header=TRUE) %>% mutate(`0.05`=R2) %>% select(gene,`0.05`)

data <- inner_join(alpha05,alpha50,by='gene')
data <- inner_join(data,alpha1,by='gene')
gdata <- gather(data,alpha,R2,2:3)
p<-ggplot(gdata, aes(y = `1` - R2, x = `1`, group=alpha, color=alpha)) + geom_point(show_guide = TRUE) + ylab(expression(paste("R"^2, " difference (LASSO - alpha)"))) + xlab(expression(paste("R"^2, " (LASSO)"))) +theme_bw(20)+ theme(legend.justification=c(0,1), legend.position=c(0,1))

tiff(filename=fig.dir %&% "Fig-DGN-EN.tiff",width=960,height=480)
multiplot(a+ggtitle('A\n')+theme(plot.title=element_text(hjust=0)),p+ggtitle('B\n')+theme(plot.title=element_text(hjust=0)),cols=2)
dev.off()

png(filename=fig.dir %&% "Fig-DGN-EN.png",width=960,height=480)
multiplot(a+ggtitle('A\n')+theme(plot.title=element_text(hjust=0)),p+ggtitle('B\n')+theme(plot.title=element_text(hjust=0)),cols=2)
dev.off()
```

##Fig 3
###DGN BSLMM
```{r dgn-bslmm, eval=TRUE}
gcta <- read.table('/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/expArch_DGN-WB_imputedGTs/DGN-WB.h2.all.models_FHSfdr0.05.all.Chr1-22_globalOtherChr.2015-03-18.txt',header=TRUE)
bslmm <- read.table(my.dir %&% 'DGN-WB_exp_BSLMM-s100K_iterations_all_genes_2015-06-14.txt',header=T)
all <- inner_join(gcta,bslmm,by='gene')
dim(all)
b<-ggplot(all,aes(x=local.h2,y=pve50))+geom_point(alpha=0.4)+coord_cartesian(xlim=c(0,1),ylim=c(0,1))+xlab(expression("GCTA h"^2))+ylab('BSLMM PVE')+geom_abline(c(0,1),color='gray')+theme_bw()
cor.test(all$local.h2,all$pve50)
data <- all %>% mutate(position=1:length(pve50),`medianSNPs<=10`=n_gamma50<=10,LCS=factor(pge025>0.01,labels=c('\u2264 0.01','> 0.01')))
a<-ggplot(data,aes(x=pve50,y=pge50,ymin=pge025,ymax=pge975,col=LCS)) + geom_pointrange(col='gray') + geom_point() + theme_bw() + xlab("PVE") + ylab("PGE") + theme(legend.position = c(1,0),legend.justification = c(1,0))

tiff(filename=fig.dir %&% "Fig-DGN-BSLMM.tiff",width=720,height=360)
multiplot(a+ggtitle('A\n')+theme(plot.title=element_text(hjust=0),text=element_text(size=18)),b + ggtitle('B\n')+ theme(plot.title=element_text(hjust=0),text=element_text(size=18)),cols=2)
dev.off()

png(filename=fig.dir %&% "Fig-DGN-BSLMM.png",width=720,height=360)
multiplot(a+ggtitle('A\n')+theme(plot.title=element_text(hjust=0),text=element_text(size=18)),b + ggtitle('B\n')+ theme(plot.title=element_text(hjust=0),text=element_text(size=18)),cols=2)
dev.off()

subdata <- select(data,pve50,pge50,`medianSNPs<=10`)
table(subdata[,3])/sum(table(subdata[,3]))
summary(subdata$pge50)

subdata <- select(data,pve50,pge50,`medianSNPs<=10`) %>% filter(pve50>0.10)
table(subdata[,3])/sum(table(subdata[,3]))
summary(subdata$pge50)

subdata <- select(data,pve50,pge50,`medianSNPs<=10`) %>% filter(pve50>0.50)
table(subdata[,3])/sum(table(subdata[,3]))
summary(subdata$pge50)
```

##Fig 4
####plot tissue-wide PVE vs GCTA marginal h2
```{r twBSh2,warning=FALSE,message=FALSE,eval=TRUE}
tislist <- scan('/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/nine.tissue.list',sep="\n",what="character")
tw <- data.frame()
rvec<-vector()
for(i in 1:length(tislist)){
  tis <- tislist[i]
  bs <- read.table(my.vol %&% tislist[i] %&% '_TW_exp_BSLMM-s100K_iterations_all_chr1-22_2015-10-18.txt',header=T,sep="\t") %>% select(gene,pve50)
  h2 <- read.table("/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/gtex-h2-estimates/GTEx.tissue-wide.h2_" %&% tislist[i] %&% "_marginal.local_2015-03-24.txt",header=T, sep="\t") %>% select(tissue,ensid,h2) %>% mutate(gene=ensid)
  explist <- scan(out.dir %&% tis %&% ".meanRPKMgt0.1_3samplesRPKMgt0_genelist","c")
  h2 <- dplyr::filter(h2,ensid %in% explist)
  subdata <- inner_join(h2,bs,by="gene")
  print(dim(subdata))
  res<-cor.test(subdata$pve50,subdata$h2)
  cat(tislist[i],"\tPearson R=",round(res$estimate,3),"\tP-value=",res$p.value,"\n")
  rvec <- cbind(rvec,unname(round(res$estimate,3)))
  tw <- rbind(tw,subdata)
}
p<-ggplot(tw,aes(x=h2,y=pve50))+geom_point(alpha=0.4)+coord_cartesian(xlim=c(-0.05,1.05),ylim=c(-0.05,1.05))+xlab(expression("GCTA h"^2))+ylab('BSLMM PVE')+geom_abline(c(0,1),color='gray') + facet_wrap(~tissue,ncol=3)+theme_bw()

ann_text <- data.frame( h2 = rep(0.75,9), pve50 = rep(0.05,9), R= rvec[1:9], tissue = factor(tislist), ensid=rep(0.9,9),gene=rep(0.9,9))
p2<-p+geom_text(data=ann_text,aes(label=paste("R==",R,sep="")),color="black",show_guide=F,parse=T,hjust=0,size=3)+ theme(legend.justification=c(1,0), legend.position=c(1,0))

png(filename=fig.dir %&% "Fig-GTEx_TW_PVE_v_h2.png",width=600,height=600)
p2
dev.off()
tiff(filename=fig.dir %&% "Fig-GTEx_TW_PVE_v_h2.tiff",width=600,height=600)
p2
dev.off()
```

##Fig 5
###Calculate entropy of Pr and correlate with **cross-tissue and tissue-specific** h2 
- higher entropy means more likely to be multi-tissue eQTL (expect positive cor. for entropy vs. cross-tissue h2)

Stephens lab data accessed from http://www.gtexportal.org/static/datasets/gtex_analysis_pilot_v3/multi_tissue_eqtls/Multi_tissue_eQTL_GTEx_Pilot_Phase_datasets.tar on 20150722.

From README: “We are using the eQTL posterior probabilities from the UC Multi-tissue eQTL method (doi:10.1371/journal.pgen.1003486) for each of the 9 tissues analyzed in the pilot phase (Adipose_subcutaneous, Artery_Tibal, Whole_Blood, Heart_Left_Ventricle, Lung, Muscle_Skeletal, Nerve_Tibial, Skin_Lower_Leg_Sun_Exposed, Thyroid) in the file res_final_uc_com_genes_com_snps.txt.gz. These values may be interpreted as Pr(SNP is eQTL in tissue s | data). 9875 eGenes are presented, with the”top" (most significant) SNP in each gene used."

```{r,entropTSh2,eval=TRUE}
mt.dir <- "/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/Multi_tissue_eQTL_GTEx_Pilot_Phase_datasets/"
h2.dir <- "/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/"
bslmm.dir <- "/Volumes/im-lab/nas40t2/hwheeler/cross-tissue/BSLMM_exp/"
mt <- read.table(mt.dir %&% "res_final_uc_com_genes_com_snps.txt.gz",header=TRUE)
#remove version number in order to compare ensembl IDs
a <- substr(mt$gene,1,15)
mt <- mutate(mt,gene=a)
h2.ts <- read.table(my.dir %&% "GTEx_Tissue-Specific_local_h2.txt",header=TRUE)
#remove version number in order to compare ensembl IDs
a <- substr(h2.ts$ensid,1,15)
h2.ts <- mutate(h2.ts,gene=a)
h2.tw <- read.table(my.dir %&% "GTEx_Tissue-Wide_local_h2.txt",header=TRUE)
#remove version number in order to compare ensembl IDs
a <- substr(h2.tw$ensid,1,15)
h2.tw <- mutate(h2.tw,gene=a)

entropy <- function(pr){
  pi <- pr/sum(pr)
  pi[pi==0] <- 1e-06 ##allows log transformation
  -1*sum(pi*log(pi))
}

mtPr <- mt[,3:11]
mtS <- apply(mtPr,1,entropy) ##calc entropy for each row
summary(mtS)
norm_mtS = (mtS-min(mtS))/(max(mtS)-min(mtS)) #normalize b/t 0 and 1
geneS <- select(mt,gene) %>% mutate(entropy=norm_mtS) ##put gene and entropy together

tis <- 'CrossTissue'

h2.tis <- h2.ts %>% select(gene,one_of(tis)) ##one_of allows character vector
newdata <- inner_join(geneS,h2.tis,by='gene')
groupdata <- dplyr::mutate(newdata, group=ifelse(entropy<0.33,0,1))
groupdata <- dplyr::mutate(newdata, group=ifelse(entropy>0.67,2,groupdata$group))
colnames(groupdata)[3] <- "h2"
p <- ggplot(groupdata,aes(y=h2,x=factor(group,labels=c("0-0.33","0.33-0.67","0.67-1"))))+geom_boxplot()+ylab(expression(paste("Cross-Tissue ", "h"^{2}))) + xlab("Uniformity")
cors <- data.frame(pval=signif(kruskal.test(h2~factor(group),groupdata)$p.value, 2))
p1 <- p + geom_text(data=cors, aes(label=paste("P = ", pval, sep="")), x=-Inf, y=Inf, hjust=-0.5, vjust=2, size=5)
print(kruskal.test(h2~factor(group),groupdata))
a <- p1 + theme_bw(16) #+ coord_cartesian(ylim=c(1.75,2.25))
print(a)

tiff(filename=fig.dir %&% "Fig-CT-entropy.tiff",width=360,height=360)
a
dev.off()

png(filename=fig.dir %&% "Fig-CT-entropy.png",width=360,height=360)
a
dev.off()
```

##Fig 6
###Table of Pr vs PVE correlations
```{r PrPVE, eval=TRUE}
pve.tw <- read.table(bslmm.dir %&% "GTEx_Tissue-Wide_local_PVE_by_BSLMM.txt",header=TRUE)
#remove version number in order to compare ensembl IDs
pve.tw <- mutate(pve.tw,gene=substr(pve.tw$gene,1,15))
mtPVE <- inner_join(mt,pve.tw,by='gene')
corMat <- round(cor(mtPVE[,3:dim(mtPVE)[2]],use="p"),3)
###Pr rows, PVE cols
mat2.tw <- corMat[1:9,11:dim(corMat)[2]]
colnames(mat2.tw)<-c('Whole: Adipose','Artery','Heart','Lung','Muscle','Nerve','Skin','Thyroid','WholeBlood')
rownames(mat2.tw)<-c('Adipose','Artery','WholeBlood','Heart','Lung','Muscle','Nerve','Skin','Thyroid')
mat2.tw <- mat2.tw[ order(row.names(mat2.tw)), ]
rownames(mat2.tw)[1] <- 'PPA: Adipose'

pve.ts <- read.table(bslmm.dir %&% "GTEx_Tissue-Specific_local_PVE_by_BSLMM.txt",header=TRUE)
#remove version number in order to compare ensembl IDs
pve.ts <- mutate(pve.ts,gene=substr(pve.ts$gene,1,15))
mtPVE <- inner_join(mt,pve.ts,by='gene')
corMat <- round(cor(mtPVE[,3:dim(mtPVE)[2]],use="p"),3)
###Pr rows, PVE cols
mat2 <- corMat[1:9,11:dim(corMat)[2]]
colnames(mat2)<-c('OTD: Adipose','Artery','Heart','Lung','Muscle','Nerve','Skin','Thyroid','WholeBlood')
rownames(mat2)<-c('Adipose','Artery','WholeBlood','Heart','Lung','Muscle','Nerve','Skin','Thyroid')
mat2 <- mat2[ order(row.names(mat2)), ]
rownames(mat2)[1] <- 'PPA: Adipose'

tiff(filename=fig.dir %&% "Fig-cor-StephensPr-v-PVE.tiff",width=960,height=480)
par(mfrow=c(1,2))
corrplot(mat2.tw,method="circle",cl.lim = c(-0.35, 0.35),is.corr = FALSE,tl.cex=1.5,cl.cex=1.2,cl.ratio = 0.4,bg='black')
corrplot(mat2,method="circle",cl.lim = c(-0.35, 0.35),is.corr = FALSE,tl.cex=1.5,cl.cex=1.2,cl.ratio = 0.4,bg='black')
dev.off()

png(filename=fig.dir %&% "Fig-cor-StephensPr-v-PVE.png",width=960,height=480)
par(mfrow=c(1,2))
corrplot(mat2.tw,method="circle",cl.lim = c(-0.35, 0.35),is.corr = FALSE,tl.cex=1.5,cl.cex=1.2,cl.ratio = 0.4,bg='black')
corrplot(mat2,method="circle",cl.lim = c(-0.35, 0.35),is.corr = FALSE,tl.cex=1.5,cl.cex=1.2,cl.ratio = 0.4,bg='black')
dev.off()
```