---
title: Genetic architecture of gene expression regulation across tissues and implications for prediction
author: Heather E. Wheeler^1,2^, GTEx Consortium, Kaanan P. Shah^3^, Nicholas Knoblauch^4^, Nancy J. Cox^5^, Dan L. Nicolae^3^, Hae Kyung Im^3^
date: '`r Sys.time()` ^1^Department of Biology and ^2^Department of Computer Science, Loyola University Chicago, ^3^Section of Genetic Medicine, Department of Medicine, University of Chicago, ^4^Committee on Genetics, Genomics, and Systems Biology, University of Chicago, ^5^Division of Genetic Medicine, Vanderbilt University'
output:
  pdf_document:
    fig_caption: yes
    keep_tex: yes
    number_sections: no
csl: plos.csl
bibliography: GenArch_references.bib
---

```{r, echo=FALSE, message=FALSE, warning=FALSE}
  # devtools::install_github("cboettig/knitcitations@v1")
  library(knitcitations); cleanbib()
  cite_options(citation_format = "pandoc", check.entries=FALSE)
  library(bibtex)
  library(ggplot2)
  library(reshape2)
  library(png)
  library(grid)
  library(kfigr)
  library(lme4)
  library(dplyr)
  library(knitr)
  library(xtable)
  "%&%" = function(a,b) paste(a,b,sep="")
  source('~/GitHub/GenArch/GenArchPaper/multiplot.R')
  fig.dir <- '~/GitHub/GenArch/GenArchPaper/Figures/'
  #BSLMM (~10 min/gene/fold with 8GB RAM and 2.2GHz AMD Opteron 6274 processor)
```

# Abstract

For many complex traits, gene regulation is likely to play a crucial mechanistic role given the consistent enrichment of expression quantitative trait loci (eQTLs) among trait-associated variants. In order to fully harness gene regulation mechanisms in complex trait prediction, we sought to better understand the underlying genetic architecture of the gene expression traits across multiple tissues. We show that local heritability of gene expression (variance in gene expression due to genetic variation within 1Mb of a gene) can be accurately estimated across tissues. However, distal heritability cannot be reliably estimated at sample sizes less than 1000, but strong functional priors can reduce the error of the distal estimates. Using both the elastic net and Bayesian sparse linear mixed modeling, we show that for local gene regulation, the genetic architecture is mostly sparse rather than polygenic. Using genome and transcriptome data from the diverse set of tissues available in The GTEx Project, we developed a model called Orthogonal Tissue Decomposition (OTD), which partitions gene expression into cross-tissue and tissue-specific components.  Estimates of the local heritability of OTD-generated cross-tissue gene expression have larger magnitude and smaller standard errors compared to single tissue estimates due to the borrowing of information across all samples. We show that genes with high cross-tissue heritability are more likely to have cross-tissue eQTLs, confirming that OTD is capturing the cross-tissue component of gene expression. 

# Introduction
Regulatory variation has been shown to play a key role in the genetics of complex traits `r citep(c("10.1371/journal.pgen.1000888","10.1371/journal.pgen.1000895","10.1016/j.ajhg.2014.10.004"))`. While many common diseases have been shown to be polygenic `r citep(c("10.1038/nature08185","10.1038/ng.2232","10.1038/ng.2383"))`, it is unclear whether gene expression levels are also polygenic or instead have simpler genetic architectures and how much these expression architectures vary across genes `r citep("10.1038/nrg3891")`. Most human expression quantitative trait loci (eQTL) studies have focused on how local genetic variation affects gene expression in order to reduce the multiple testing burden that would be required for a global analysis `r citep(c("10.1371/journal.pgen.1002639","10.1038/nrg3891"))`. Furthermore, when both local and distal eQTLs are reported `r citep(c("10.1038/ng2142","10.1371/journal.pgen.1002078","10.1038/ng.2951"))`, effect sizes and replicability are much higher for local eQTLs. Indeed, while the heritability of gene expression attributable to local genetic variation has been estimated accurately, large standard errors have prevented accurate estimation of the contribution of distal genetic variation to gene expression variation `r citep(c("10.1371/journal.pgen.1001317", "10.1038/ng.2951"))`.

We assessed the ability of various models with different underlying assumptions to predict gene expression in order to both understand the underlying genetic architecture of gene expression and to further optimize predictors for our complex trait prediciton method PrediXcan `r citep("10.1038/ng.3367")`. In our PrediXcan paper, we showed that a polygenic score model was suboptimal to more sparse models like the elastic net model with $\alpha=0.5$. However, a top eQTL only model did not do as well as the elastic net either `r citep("10.1038/ng.3367")`, suggesting that for many genes, the genetic architecture is sparse, but not regulated by one SNP. Gene expression traits with sparse architecture should be better predicted with sparse models such as LASSO (Least Absolute Shrinkage and Selection Operator) `r citep("http://www.jstor.org/stable/2346178")` while highly polygenic traits should be better predicted with ridge regression  or similarly polygenic models `r citep(c("10.1080/00401706.1970.10488635","10.1038/nrg2898","10.1002/gepi.21808"))`. To obtain a more thorough understanding of gene expression achitecture, we used the elastic net and the hybrid approach of BSLMM (Bayesian Sparse Linear Mixed Model) `r citep("10.1371/journal.pgen.1003264")` to quantify sparse and polygenic effects.

Most human eQTL studies have been performed in whole blood or lymphoblastoid cell lines due to ease of access or culturabilty `r citep(c("10.1038/nature04244","10.1038/ng2142","10.1101/gr.155192.113"))`. The Genotype-Tissue Expression (GTEx) Project aims to examine the genetics of gene expression more comprehensively and recently published a pilot analysis of eQTL data from 1641 samples across 43 tissues from 175 individuals confirming that eQTLs are highly shared across tissues `r citep("10.1126/science.1262110")`. We have shown that gene expression predictors perform well across tissues `r citep("10.1038/ng.3367")`. In order to harness this cross-tissue effect for prediction and to better understand the genetic architecture of tissue-specific and cross-tissue gene regulation, we developed a mixed effects model called orthogonal tissue decomposition (OTD) to determine the cross-tissue and tissue-specific components of gene expression in the rich GTEx dataset. We modeled the underlying genetic architecture of the cross-tissue and tissue-specific gene expression components and developed predictors for use in PrediXcan `r citep("10.1038/ng.3367")`.

Specifically, in this paper we show that local heritability can be accurately estimated across tissues, but distal heritability cannot reliably estimated at current sample sizes. Yet, strong functional priors reduce distal heritability error. We also show that for local gene regulation, the genetic architecture is mostly sparse rather than polygenic. We found that estimates of the local heritability of cross-tissue gene expression generated with our OTD model have larger magnitude and improved standard errors compared to single tissue estimates due to the borrowing of information across all samples. Comparing our OTD results to a previously performed joint multi-tissue eQTL analysis method `r citep("10.1371/journal.pgen.1003486")`, we show that genes with high cross-tissue heritability are more likely to have cross-tissue eQTLs, confirming that OTD is capturing the cross-tissue component of gene expression. Cross-validated predictors of cross-tissue and tissue-specific expression built here have been added to the PrediXcan database for future investigations of complex traits.

# Results

## Local genetic variation explains a large proportion of gene expression variance
We estimated the heritability of gene expression in whole blood from the Depression Genes and Networks (DGN) cohort (n=922) `r citep("10.1101/gr.155192.113")` using a mixed-effects model (see Methods) and calculated variances using restricted maximum likelihood as implemented in GCTA `r citep("10.1016/j.ajhg.2010.11.011")`. We fit two joint models, each with the same local genetic relationship matrix (GRM) and a two different distal GRMs. The local GRM was derived from SNPs within 1 Mb of each gene in both models. In model 1, the distal GRM was derived from all SNPs on non-gene chromosomes. In model 2, the distal GRM was derived from SNPs that are located on non-gene chromosomes and are eQTLs in the Framingham Heart Study (FHS) cohort (n=5257, FDR < 0.05) `r citep("10.1038/ng.3220")`. The mean local h^2^ was similar in both models, 0.123 in model 1 and 0.130 in model 2 (`r figr("jointH2", prefix=TRUE, link=TRUE, type="Fig")`). Of expressed genes, 52.6\% and 54.6\% had a positive 95\% confidence interval (CI) for the local estimate in models 1 and 2, respectively. The mean distal h^2^ was 0.123 with a standard error (SE) of 0.284 in model 1, while the mean distal h^2^ was 0.076 with a SE of 0.142 in model 2. Using functional priors (known eQTLs) to define distal h^2^ in model 2 increased the percentage of genes with a positive CI from 3.2\% to 4.2\% (`r figr("jointH2", prefix=TRUE, link=TRUE, type="Fig")`). In model 2, the maximum local h^2^ was 0.93 with a standard error (SE) of 0.009 while the maximum distal h^2^ was 0.91 with a SE of 0.16. 

We also estimated the heritability of gene expression in 9 tissues from the GTEx Project; sample sizes ranged from 190-361 (see Methods). Mean local h^2^ estimates ranged from 0.028-0.044 with 4.4-8.6\% of genes having a positive CI (`r figr("TWlocalh2", prefix=TRUE, link=TRUE, type="Fig")`)). While the reduced sample sizes reduced the overall mean, maximum local h^2^ estimates in each tissue ranged from 0.79-0.91, similar to the maximum seen in DGN. As expected, distal h^2^ (eQTLs on non-gene chromosomes) could not be reliably estimated in these smaller sample sizes as less than 1\% of genes in each tissue had a positive CI in the joint model (`r figr("TWglojth2", prefix=TRUE, link=TRUE, type="Fig")`-**[SUP]**)).

[[[We found no correlation between expression levels and local h^2^ estimates in DGN [[make SUP fig]].
[[TO DO: plot a) array based h2 vs. RPKM in gtex, use alkes and or fred wright's h2 estimates b) RNAseq h2 vs RPKM in gtex]]---> h2 (from Price, Wright,& us) and GTEx RPKM do correlate, conflicting results b/t GTEx and DGN. Maybe b/c DGN is raw counts, not RPKM? remove from paper?]]]

## The effect of local genetic variation on gene expression is sparse rather than polygenic
We performed 10-fold cross-validation using the elastic net `r citep("10.1111/j.1467-9868.2005.00503.x")` to test the predictive performance of local SNPs for gene expression across a range of mixing parameters, $\alpha$. The $\alpha$ that gives the largest cross-validation R^2^ informs the sparsity of each gene expression trait. That is, at one extreme, if the optimal $\alpha=0$ (equivalent to ridge regression), the gene expression trait is highly polygenic, whereas if the optimal $\alpha=1$ (equivalent to LASSO), the trait is highly sparse. We found that for most gene expression traits, the cross-validated R^2^ was suboptimal for $\alpha=0$ and $\alpha=0.05$, but nearly identically optimal for $\alpha=0.5$ through $\alpha=1$ in the DGN cohort (`r figr("EN",prefix=TRUE,link=TRUE,type="Fig")`). An $\alpha=0.05$ was also clearly suboptimal for gene expression prediction in the nine GTEx tissues, while models with $\alpha=0.5,0.95,1$ had similar predictive power (`r figr("ENtw",prefix=TRUE,link=TRUE,type="Fig")`-**[SUP]**). This suggests that for most genes, the effect of local genetic variation on gene expression is sparse rather than polygenic.

To further examine sparsity and polygenicity, we used BSLMM `r citep("10.1371/journal.pgen.1003264")` to define the total proportion of variance in expression explained by sparse and polygenic effects together (PVE) and the proportion of this genetic variance explained by sparse effects (PGE) for local SNPs in each gene in the DGN cohort. The PVE can be thought of as a Bayesian estimate of chip heritability and, indeed, there is a strong correlation between BSLMM-estimated PVE and GCTA-estimated h^2^ (`r figr("dgnBSLMM",prefix=TRUE,link=TRUE,type="Fig")`A, R=0.96). For genes with large PVE, the PGE also was large, indicative of a sparse genetic architecture. For example, all genes with PVE > 0.50 had PGE > 0.82 and their median PGE was 0.989 (`r figr("dgnBSLMM",prefix=TRUE,link=TRUE,type="Fig")`B). The median PGE for genes with PVE > 0.1 was 0.949. Fittingly, for most (96.3\%) of the genes with PVE estimates > 0.10, the median number of SNPs included in the model was no more than 10 (`r figr("dgnBSLMM",prefix=TRUE,link=TRUE,type="Fig")`C).

Interestingly, when we applied BSLMM to the GTEx data, we found that many genes had strikingly larger BSLMM-estimated PVE than GCTA-estimated h^2^ (`r figr("TWpveh2",prefix=TRUE,link=TRUE,type="Fig")`). This likely reflects the increased power of the BSLMM method at the lower sample sizes (n<=361) present in GTEx to estimate variance explained when the trait is more sparse than polygenic. GCTA assumes an underlying polygenic model, but as we saw in DGN, BSLMM-estimated PVE and GCTA-estimated h^2^ are more correlated when the sample size is larger (n=922, `r figr("dgnBSLMM",prefix=TRUE,link=TRUE,type="Fig")`A). As we observed in DGN, genes with larger PVE estimates were more likely to have a PGE estimate approaching 1 with a lower credible set greater than 0.01 in each of the nine GTEx tissues (`r figr("TWbslmm",prefix=TRUE,link=TRUE,type="Fig")`). 

## Cross-tissue and tissue-specific gene expression by orthogonal tissue decomposition
In order to better understand the context specificity of gene expression regulation, we developed a method called orthogonal tissue decomposition (OTD), which uses a mixed effects model to generate cross-tissue and tissue-specific gene expression levels (see Methods). Using a marginal model with just the local GRM, we estimated the local h^2^ of cross-tissue gene expression and tissue-specific gene expression in the nine tissues with the most samples. The cross-tissue heritabilities were larger and the standard errors were smaller than the tissue-specific estimates (`r figr("TSotdH2SE", prefix=TRUE, link=TRUE, type="Fig")`-**[SUP]**). The percentage of GCTA h^2^ estimates with positive CIs was much larger for cross-tissue expression (17.3\%) than the tissue-specific expressions (all less than 3\%, `r figr("otdTSh2", prefix=TRUE, link=TRUE, type="Fig")`). Similarly, the percentage of BSLMM PVE estimates with a lower credible set greater than 0.01 was 49\% for cross-tissue expression, but ranged from 24-27\% for tissue-specific expression (`r figr("CTTSbslmm", prefix=TRUE, link=TRUE, type="Fig")`).

We also compared the cross-tissue h^2^ from the OTD to h^2^ estimates from the pre-OTD measures of gene expression in each of the nine tissues, which we term whole tissue expression. Again, the cross-tissue heritabilities were larger and the standard errors were smaller than the whole tissue estimates (`r figr("TWotdH2SE", prefix=TRUE, link=TRUE, type="Fig")`-**[SUP]**), though less striking than the tissue-specific comparison. The percentage of whole tissue h^2^ estimates with positive CIs ranged from 4.4-8.6\% and thus were all larger than the tissue-specific postive CI percentages, but smaller than the cross-tissue percentage (`r figr("TWlocalh2", prefix=TRUE, link=TRUE, type="Fig")`). Cross-tissue BSLMM PVE estimates had lower error than whole tissue PVE (`r figr("TWbslmm",prefix=TRUE,link=TRUE,type="Fig")`, `r figr("CTTSbslmm", prefix=TRUE, link=TRUE, type="Fig")`). Like whole tissue expression, cross-tissue and tissue-specific expression showed better predictive performance using the elastic-net when $\alpha=0.5$ - $\alpha=1$ than when $\alpha=0.05$ (`r figr("ENctts",prefix=TRUE,link=TRUE,type="Fig")`-**[SUP]**). Cross-tissue predictive performance exceeded that of both tissue-specific and whole tissue expression as indicated by higher cross-validated R^2^ (`r figr("ENtw",prefix=TRUE,link=TRUE,type="Fig")`-**[SUP]**, `r figr("ENctts",prefix=TRUE,link=TRUE,type="Fig")`-**[SUP]**).

We compared our OTD results to those from a joint multi-tissue eQTL analysis method `r citep("10.1371/journal.pgen.1003486")`, which was previously performed on a subset of the GTEx data `r citep("10.1126/science.1262110")`. The results of this analysis include eQTL posterior probabilities for each of the nine tissues, which can be interpreted as the probability a SNP is an eQTL in tissue *x* given the data. Using the top eQTL for each gene, we defined an entropy statistic (see Methods) that combines the nine posterior probabilities into one value such that higher entropy values mean the gene is more likely to be regulated by the same eQTL across all nine tissues, rather than in just a subset of the nine. We observed a strong correlation between entropy and both the cross-tissue expression heritability (R = 0.082, `r figr("entropyCT", prefix=TRUE, link=TRUE, type="Fig")`A) and PVE (R = 0.12, `r figr("entropyCT", prefix=TRUE, link=TRUE, type="Fig")`B) estimates, using the cross-tissue expression derived from the OTD. Thus, genes with high cross-tissue heritability are more likely to have cross-tissue eQTLs, confirming that OTD is capturing the cross-tissue component of gene expression. Also, the correlation between tissue-specific OTD gene expression PVE and the posterior probability that the gene has an eQTL in that tissue is strongest in each respective tissue, confirming that OTD also captures tissue-specific components of gene expression (`r figr("corrplot", prefix=TRUE, link=TRUE, type="Fig")`). 

# Discussion
Because regulatory variation plays a key mechanistic role in the genetics of complex traits `r citep(c("10.1371/journal.pgen.1000888","10.1371/journal.pgen.1000895","10.1016/j.ajhg.2014.10.004"))`, we sought to comprehensively characterize the genetic architecture of gene expression across tissues. We accurately quantify the local heritability of gene expression in DGN whole blood and nine GTEx tissues. In DGN, the mean local h^2^ was 0.13, similar to that found in family studies of blood expression, where mean h^2^ ranged from 0.07-0.11 `r citep(c("10.1371/journal.pgen.1001317", "10.1038/ng.2951"))`. While we found that functional priors (known trans-eQTLs) can reduce the error of the estimate by reducing the number of genetic markers included in the genetic relationship matrix, larger sample sizes (n > 1000) are needed to accurately estimate distal heritability. 

Using the hybrid polygenic-sparse approach of BSLMM (Bayesian Sparse Linear Mixed Model) `r citep("10.1371/journal.pgen.1003264")`, we show that the local architecture of gene expression is sparse (high PGE) for most heritable genes in both DGN and GTEx. Using the elastic net `r citep("10.1111/j.1467-9868.2005.00503.x")`, we observed improved cross-validated expression prediction for $\alpha>=0.5$ across tissues, confirming the sparsity result. Our BSLMM analysis quantified the optimal number of SNPs to include for each gene. For example, the median number of SNPs for *ERAP2* was 1. When we previously plotted out-of-sample observed vs. predicted expression using elastic net ($\alpha=0.5$) generated predictors for this gene, we saw three clusters, corresponding to each of the three genotypes for the causal variant `r citep("10.1038/ng.3367")`. Similarly, BSLMM estimated 2, 3, and 5 SNPs for *PEX6*, *NUDT2*, and *ERAP1*, respectively, consistent with the out-of-sample observed vs. predicted expression plots in our PrediXcan paper (see Figure 5 in `r citep("10.1038/ng.3367")`). Potentially due to variations in imputation quality of the input SNPs for expression prediction, it is useful to include more than the likely causal variants (i.e. elastic net rathern than BSLMM) in the prediction for robustness. In addition, the elastic net is amenable cross-validation, while genome-wide cross-validation with BSLMM is impractical at current runtimes.

We developed a mixed effects model called orthogonal tissue decomposition (OTD) to determine the cross-tissue and tissue-specific components of gene expression in the GTEx dataset. Previous studys have shown that many eQTLs are shared across tissues `r citep(c("10.1126/science.1262110","10.1371/journal.pgen.1003486"))`. In addition, because expression data from multiple tissues were available from the same individuals in GTEx, we could effectively use the multiple tissue samples as subject replicates in our OTD model. However, the tissue availability is unbalanced across individuals because of the difficulties of sample collection and the uneven quality of the tissues. By combining all available expression data in our OTD model, we found that estimates of the local heritability of cross-tissue gene expression have larger magnitude and improved standard errors compared to single tissue estimates due to the borrowing of information across all samples. Thus, OTD effectively increases power to estimate heritability. Comparing our OTD results to a previously performed joint multi-tissue eQTL analysis method `r citep("10.1371/journal.pgen.1003486")`, we show that genes with high cross-tissue heritability are more likely to have cross-tissue eQTLs, confirming that OTD is capturing the cross-tissue component of gene expression.

In this paper, we quantitate the genetic architecture of gene expression and develop predictors across tissues. We show that local heritability can be accurately estimated across tissues, but distal heritability cannot be reliably estimated at current sample sizes. Using two different approaches, the elastic net and BSLMM, we show that for local gene regulation, the genetic architecture is mostly sparse rather than polygenic. Using new expression phenotypes generated in our OTD model, we show that cross-tissue predictive performance exceeded that of both tissue-specific and whole tissue expression as indicated by higher elastic net cross-validated R^2^. Predictors generated in this study of gene expression architecture have been added to our PrediXcan database (<https://github.com/hakyimlab/PrediXcan>) `r citep("10.1038/ng.3367")` for use in future studies of complex trait genetics. 

# Methods

## Genomic and Transcriptomic Data

### DGN Dataset
We obtained whole blood RNA-Seq and genome-wide genotype data for 922 individuals from the Depression Genes and Networks (DGN) cohort `r citep("10.1101/gr.155192.113")`, all of European ancestry. For our analyses, we used the HCP (hidden covariates with prior) normalized gene-level expression data used for the *trans*-eQTL analysis in Battle et al. `r citep("10.1101/gr.155192.113")` and downloaded from the NIMH repository. The 922 individuals were unrelated (all pairwise $\hat{\pi}$ < 0.05) and thus all included in downstream analyses. Imputation of approximately 650K input SNPs (minor allele frequency [MAF] > 0.05, Hardy-Weinberg Equilibrium [P > 0.05], non-ambiguous strand [no A/T or C/G SNPs]) was performed on the University of Michigan Imputation-Server (<https://imputationserver.sph.umich.edu/start.html>) `r citep(c("10.1038/ng.2354","10.1093/bioinformatics/btu704"))` with the following parameters: 1000G Phase 1 v3 ShapeIt2 (no singletons) reference panel, SHAPEIT phasing, and EUR population. Approximately 1.9M non-ambiguous strand SNPs with MAF > 0.05, imputation R^2^ > 0.8 and, to reduce computational burden, inclusion in HapMap Phase II were retained for subsequent analyses.

### GTEx Dataset
We obtained RNA-Seq gene expression levels from 8555 tissue samples (53 unique tissue types) from 544 unique subjects in the GTEx Project `r citep("10.1126/science.1262110")` data release on 2014-06-13. Of the individuals with gene expression data, genome-wide genotypes (imputed with 1000 Genomes) were available for 450 individuals. While all 8555 tissue samples were used in the OTD model (described below) to generate cross-tissue and tissue-specific components of gene expression, we used the nine tissues with the largest sample sizes when quantifying tissue-specific effects. Tissues and sample sizes (both RNA-seq and genotypes available) included cross-tissue ($n=450$), skeletal muscle ($n=361$), whole blood ($n=339$), skin from the sun-exposed portion of the lower leg ($n=303$), subcutaneous adipose ($n=298$), tibial artery ($n=285$), lung ($n=279$), thyroid ($n=279$), tibial nerve ($n=256$) and left ventricle heart ($n=190$). Approximately 2.6M non-ambiguous strand SNPs included in HapMap Phase II were retained for subsequent analyses.

## Partitioning local and distal heritability of gene expression
To investigate the proximity of gene expression regulation to each gene, we partitioned the proportion of gene expression variance explained by SNPs in the DGN cohort into two components: local (SNPs within 1Mb of the gene) and distal (eQTLs on non-gene chromosomes) as defined by the GENCODE `r citep("10.1101/gr.135350.111")` version 12 gene annotation. We calculated the proportion of the variance (narrow-sense heritability) explained by each component using the following mixed-effects model:

$$ Y_g = \sum_{k = \in local}w_{k,g} X_k + \sum_{k = \in distal}w_{k,g} X_k + \epsilon $$ 

Assuming a random effects for $w_{k,g} \approx N(0, \sigma^2_w)$ and $\epsilon \approx N(0, \sigma^2_{\epsilon} I_n)$, where $I_n$ is the identity matrix, we calculated the total variability explained by local and distal components by estimating $\sigma^2_w$ with restricted maximum likelihood (REML) using GCTA software `r citep("10.1016/j.ajhg.2010.11.011")`. For heritability analyses in the GTEx cohort, we removed the $distal$ term from the model and only estimated marginal $local$ h^2^ due to the smaller sample sizes of both cross-tissue and tissue-specific expression levels compared to DGN.

## Determining polygenicity versus sparsity using the elastic net
We applied the elastic net `r citep("10.1111/j.1467-9868.2005.00503.x")` to model the effect of local genetic variation (SNPs within 1 Mb of gene) on the genetic architecture of gene expression. We used the `cv.glmnet` function in the R package `glmnet` `r citep(citation("glmnet"))` to perform 10-fold cross-validation of the elastic net across a range of mixing paramaters ($\alpha$) to find the $\alpha$ that maximized predictive performance, measured by Pearson's R^2^. Specifically, `glmnet` solves the following problem:

**[[Haky, should/can we simplify this equation? I just took it from <http://web.stanford.edu/~hastie/glmnet/glmnet_alpha.html>, but they don't explain all the terms]]**
<span class="math">\[\min_{\beta_0,\beta} \frac{1}{N} \sum_{i=1}^{N} w_i l(y_i,\beta_0+\beta^T x_i) + \lambda\left[(1-\alpha)||\beta||_2^2/2 + \alpha ||\beta||_1\right], \]</span>

over a grid of values of $\lambda$ covering the entire range `r citep(citation("glmnet"))`. This tuning parameter $\lambda$ controls the overall strength of the penalty. 

The elastic net penalty is controlled by mixing parameter $\alpha$, which spans LASSO ($\alpha=1$, the default) `r citep("http://www.jstor.org/stable/2346178")` at one extreme and ridge regression ($\alpha=0$) `r citep("10.1080/00401706.1970.10488635")` at the other. The ridge penalty shrinks the coefficients of correlated SNPs towards each other, while the LASSO tends to pick one of the correlated SNPs and discard the others. Thus, an optimal prediction R^2^ for $\alpha=0$ means the gene expression trait is highly polygenic, while an optimal prediction R^2^ for $\alpha=1$ means the trait is highly sparse. An optimal prediction R^2^ in between (e.g. $\alpha=0.5$) means the trait has a mixed genetic architecture.

In the DGN cohort, we tested 21 values of the mixing parameter ($\alpha=0, 0.05, 0.1, ..., 0.90, 0.95, 1$) for optimal prediction of gene expression of the 341 genes on chromosome 22. For the rest of the autosomes in DGN and for whole tissue, cross-tissue, and tissue-specific expression in the GTEx cohort, we tested $\alpha=0.05, 0.5, 0.95, 1$.

## Quantifying sparsity with Bayesian Sparse Linear Mixed Models (BSLMM)
We used BSLMM `r citep("10.1371/journal.pgen.1003264")` to model the effect of local genetic variation (SNPs within 1 Mb of gene) on the genetic architecture of gene expression. The BSLMM consists of a standard linear mixed model, with one random effect term, and with sparsity inducing priors on the regression coefficients `r citep("10.1371/journal.pgen.1003264")`. We used the software GEMMA `r citep("10.1038/ng.2310")` to implement BSLMM for each gene using the following parameters:

```{r, eval=FALSE}
gemma -g [localGenoFile] -p [geneExpFile] -a [snpAnnotFile] -bslmm 1 -s 100000 -o [outFile]
```

The `-bslmm 1` option specifies a linear BSLMM and the `-s 100000` option specifies the number of sampling steps per gene. The BSLMM estimates the PVE (the total proportion of variance in phenotype explained by the sparse effects and random effects terms together) and PGE (the proportion of genetic variance explained by the sparse effects terms). From the second half of the sampling iterations for each gene, we report the median and the 95\% credible sets of the PVE, PGE, and the |$\gamma$| parameter (the number of SNPs with non-zero coefficients).

## Orthogonal tissue decomposition
To better understand the context specificity of gene expression regulation, we developed a method called orthogonal tissue decomposition (OTD). This approach is an extension of our method to develop an intrinsic growth phenotype `r citep("10.1371/journal.pgen.1002525")`. We applied OTD to GTEx Project `r citep("10.1126/science.1262110")` data and decomposed the expression of each gene into cross-tissue and tissue-specific components. The tissue availability is unbalanced across individuals because of the difficulties of sample collection and the uneven quality of the tissues.  OTD decomposes the expression traits into orthogonal components as represented by the following model:

$$ Y_i = T_{i,cross} + T_{i,tissue} $$

Specifically, to generate cross-tissue and tissue-specific expression levels, we used the `lmer` function in the R `r citep(citation())` package `lme4` `r citep(citation("lme4"))` to fit the following mixed-effects model:

```{r, eval=FALSE}
fit <- lme4::lmer(expression ~ (1|SUBJID) + TISSUE + GENDER + PEERs)
```

The model included whole tissue gene expression levels in 8555 GTEx tissue samples from 544 unique subjects. A total of 17,647 Protein-coding genes (defined by GENCODE `r citep("10.1101/gr.135350.111")` version 18) with a mean gene expression level across tissues greater than 0.1 RPKM (reads per kilobase of transcript per million reads mapped) were included in the model. `SUBJID` was a random effect and the covariates `TISSUE`, `GENDER`, and `PEERs` were fixed effects used to predict whole tissue expression levels (`expression` in the model). `PEERs` included the top 15 PEER factors estimated across all tissues using the R package `PEER` `r citep("10.1038/nprot.2011.457")` to control for batch effects and experimental confounders. Cross-tissue expression was defined as the random effects from the model (`ranef(fit)`) and tissue-specific expression as the residuals (`resid(fit`)).

## Comparison of OTD PVE to multi-tissue eQTL results

Using results from a joint multi-tissue eQTL analysis method `r citep("10.1371/journal.pgen.1003486")` performed with a subset of the GTEx data (maximum n=175 in the nine tissues of the pilot phase, see `r citep("10.1126/science.1262110")`), we defined an entropy statistic to compare these results to those from our OTD method. The results of the multi-tissue analysis include eQTL posterior probabilities for each of the nine tissues, which can be interpreted as the probability a SNP is an eQTL in tissue $t$ given the data. Using the top eQTL for each gene $g$, we defined the entropy $S_g$ as:

$$ S_g = -\sum_{t}p_{t,g} \log p_{t,g} $$

where $p_{t,g}$ is the eQTL probability in tissue $t$ normalized to 1 for each gene $g$. Thus, eQTLs with higher entropy statistics are more likely to be cross-tissue eQTLs, rather than only regulating gene expression in one or a few tissues. We calculated the Pearson correlation between $S_g$ and the cross-tissue expression heritability and PVE for each gene to verify that our OTD method captures cross-tissue effects. We also calculated a Pearson correlation matrix between the posterior probabilities in each tissue from the multi-tissue eQTL method and the tissue-specific gene expression PVE from the OTD method.


## Tables

```{r kable, echo=FALSE, eval=FALSE, message=FALSE}
library(knitr)

x <- rnorm(100)
y <- 2*x + rnorm(100)
out <- lm(y ~ x)
kable(summary(out)$coef, digits=2, caption="This is a GLM summary table.")
```

```{r PrPVE,message=FALSE,echo=FALSE,eval=FALSE}
tab <- read.table(fig.dir %&% 'Table-cor-StephensPr-OTDtsPVE.txt',header=TRUE,sep="\t",as.is=TRUE)
colnames(tab)[1] <- "OTD: Adipose"
kable(tab, caption="Pearson correlation between the posterior probability the top multi-tissue eQTL regulates its gene in a given tissue (Flutre et al. method) and the PVE of tissue-specific gene expression from the orthogonal tissue decomposition (OTD).")
```

```{r xtable, results="asis", echo=FALSE, eval=FALSE, message=FALSE}
library(xtable)
tab <- xtable(summary(out)$coef, digits=c(0, 2, 2, 1, 2), 
              caption="This is a GLM summary table.")
print(tab, type="latex")
```

# Figures

```{r jointH2, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="DGN whole blood expression joint heritability (h^2^). Local (SNPs within 1 Mb of each gene) and distal (Left: SNPs on non-gene chromsomes. Right: SNPs that are eQTLs in the Framingham Heart Study on other chromosomes [FDR < 0.05]) h^2^ for gene expression were jointly estimated. (**Top**) Distal h^2^ compared to local h^2^ per gene in each model. (**Middle**) Local and (**Bottom**) distal gene expression h^2^ estimates ordered by increasing h^2^. The 95% confidence interval (CI) of each h^2^ estimate is in gray and genes with a lower bound greater than zero are in blue.",fig.width=4,fig.height=8}
fig1<-readPNG(fig.dir %&% 'Fig-DGN-jt-h2.png')
grid.raster(fig1)
```

```{r TWlocalh2,echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx whole tissue local heritability (h^2^) estimation. Local (SNPs within 1 Mb of each gene) gene expression h^2^ estimates ordered by increasing h^2^. The 95% confidence interval (CI) of each h^2^ estimate is in gray and genes with a lower bound greater than zero are in blue",fig.width=6,fig.height=4}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_TW_local_h2.png')
grid.raster(fig)
```

```{r TWglojth2,echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx whole tissue distal heritability (h^2^) estimation. Distal (SNPs that are eQTLs in the Framingham Heart Study on other chromosomes [FDR < 0.05]) gene expression h^2^ estimates from a joint model are ordered by increasing h^2^. The 95% confidence interval (CI) of each h^2^ estimate is in gray and genes with a lower bound greater than zero are in blue",fig.width=6,fig.height=4}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_TW_glo.jt.h2.png')
grid.raster(fig)
```

```{r EN,echo=FALSE, warning=FALSE, message=FALSE, fig.cap="DGN cross-validated predictive performance across the elastic net. (**A**) 10-fold cross-validated R^2^ of predicted vs. observed expression in DGN whole blood compared to a range of elastic net mixing parameters ($\\alpha$) for genes on chromosome 22 with R^2^ > 0.3. (**B**) Predictive R^2^ difference between LASSO ($\\alpha = 1$) and several other values of $\\alpha$ compared to LASSO predictive R^2^ for 13171 autosomal genes."}
fig3<-readPNG(fig.dir %&% 'Fig-DGN-EN.png')
grid.raster(fig3)
```

```{r ENtw,echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx whole tissue cross-validated predictive performance across the elastic net. Predictive R^2^ difference between LASSO ($\\alpha = 1$) and several other values of $\\alpha$ compared to LASSO predictive R^2^ for 18473 autosomal genes per tissue."}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_TW_EN_CV.png')
grid.raster(fig)
```

```{r dgnBSLMM,echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Bayesian Sparse Linear Mixed Models reveal the sparsity of gene expression architecture. (**A**) BSLMM-estimated PVE (total proportion of variance explained) compared to GCTA-estimated heritability per gene (R=0.96) (**B**) Comparison of median PGE (proportion of PVE explained by sparse effects) to median PVE (total proportion of variance explained) for expression of each gene. The 95% credible set of each PGE estimate is in gray and genes with a lower credible set (LCS) greater than 0.01 are in blue. (**C**) Comparison of the median number of SNPs included in the model of each gene to median PVE. The 95% credible set of each SNP-number estimate is in gray and genes with a median of 10 or fewer SNPs are in blue.",fig.width=3,fig.height=8}
fig<-readPNG(fig.dir %&% 'Fig-DGN-BSLMM.png')
grid.raster(fig)
```

```{r TWpveh2, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx whole tissue expression BSLMM-estimated PVE (total proportion of variance explained) compared to GCTA-estimated heritability per gene. R = Pearson correlation."}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_TW_PVE_v_h2.png')
grid.raster(fig)
```

```{r TWbslmm, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx whole tissue expression comparison of median PGE (proportion of PVE explained by sparse effects) to median PVE (total proportion of variance explained) for expression of each gene. The 95% credible set of each PGE estimate is in gray and genes with a lower credible set (LCS) greater than 0.01 are in blue."}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_TW_BSLMM.png')
grid.raster(fig)
```

```{r TSotdH2SE, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Cross-tissue and tissue-specific comparison of heritability (h^2^, **A**) and standard error (SE, **B**) estimation.  Cross-tissue local h^2^ is estimated using the cross-tissue component (random effects) of the mixed effects model for gene expression and SNPs within 1 Mb of each gene. Tissue-specifc local h^2^ is estimated using the tissue-specific component (residuals) of the mixed effects model for gene expression for each respective tissue and SNPs within 1 Mb of each gene.",fig.width=4,fig.height=8}
figS2<-readPNG(fig.dir %&% 'FigS2.png')
grid.raster(figS2)
```

```{r otdTSh2, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Cross-tissue heritability (h^2^) compared to tissue-specific h^2^.  Cross-tissue local h^2^ is estimated using the cross-tissue component (random effects) of the mixed effects model for gene expression and SNPs within 1 Mb of each gene. Tissue-specifc local h^2^ is estimated using the tissue-specific component (residuals) of the mixed effects model for gene expression for each respective tissue and SNPs within 1 Mb of each gene.",fig.width=5,fig.height=8}
figS1<-readPNG(fig.dir %&% 'Fig-GTEx_CT-TS_local_h2.png')
grid.raster(figS1)
```

```{r CTTSbslmm, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx orthogonal tissue decomposition cross-tissue and tissue-specific expression comparison of median PGE (proportion of PVE explained by sparse effects) to median PVE (total proportion of variance explained) for expression of each gene. The 95% credible set of each PGE estimate is in gray and genes with a lower credible set (LCS) greater than 0.01 are in blue."}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_CT-TS_BSLMM.png')
grid.raster(fig)
```

```{r TWotdH2SE, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Cross-tissue and whole tissue comparison of heritability (h^2^, **A**) and standard error (SE, **B**).  Cross-tissue local h^2^ is estimated using the cross-tissue component (random effects) of the mixed effects model for gene expression and SNPs within 1 Mb of each gene. Whole tissue local h^2^ is estimated using the measured gene expression for each respective tissue and SNPs within 1 Mb of each gene.",fig.width=4,fig.height=8}
fig5<-readPNG(fig.dir %&% 'Fig5.png')
grid.raster(fig5)
```

```{r ENctts,echo=FALSE, warning=FALSE, message=FALSE, fig.cap="GTEx orthogonal tissue decomposition cross-tissue and tissue-specific expression cross-validated predictive performance across the elastic net. Predictive R^2^ difference between LASSO ($\\alpha = 1$) and several other values of $\\alpha$ compared to LASSO predictive R^2^ for 18473 autosomal genes per tissue."}
fig<-readPNG(fig.dir %&% 'Fig-GTEx_CT-TS_EN_CV.png')
grid.raster(fig)
```

```{r entropyCT, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Entropy of the posterior probabilities from the Flutre et al. multi-tissue eQTL method compared to the estimates of (**A**) heritability and (**B**) PVE of cross-tissue gene expression derived from the orthogonal tissue decomposition. The generalized additive model smoothing line is in red."}
fig <- readPNG(fig.dir %&% 'Fig-CT-entropy.png')
grid.raster(fig)
```

```{r corrplot, echo=FALSE, warning=FALSE, message=FALSE, fig.cap="Pearson correlation (R) between the posterior probability the top multi-tissue eQTL regulates its gene in a given tissue (eQTL, Flutre et al. method) and the PVE of tissue-specific gene expression from the orthogonal tissue decomposition (OTD). Area of each circle is proportional to the absolute value of R."}
fig <- readPNG(fig.dir %&% 'Fig-cor-StephensPr-OTDtsPVE.png')
grid.raster(fig)
```

#Supplemental Figures




# References

```{r, warning=FALSE, message=FALSE, echo=FALSE} 
  write.bibtex(file="GenArch_references.bib")
```




