\documentclass[]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
    \usepackage{xltxtra,xunicode}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
  \newcommand{\euro}{â‚¬}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1in]{geometry}
\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{{#1}}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{{#1}}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{{#1}}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{{#1}}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{{#1}}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{{#1}}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{{#1}}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{{#1}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{{#1}}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{{#1}}}
\newcommand{\RegionMarkerTok}[1]{{#1}}
\newcommand{\ErrorTok}[1]{\textbf{{#1}}}
\newcommand{\NormalTok}[1]{{#1}}
\usepackage{longtable,booktabs}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\ifxetex
  \usepackage[setpagesize=false, % page size defined by xetex
              unicode=false, % unicode breaks when used with xetex
              xetex]{hyperref}
\else
  \usepackage[unicode=true]{hyperref}
\fi
\hypersetup{breaklinks=true,
            bookmarks=true,
            pdfauthor={Heather E. Wheeler1, Nicholas Knoblauch2, GTEx Consortium, Nancy J. Cox3, Dan L. Nicolae1, Hae Kyung Im1},
            pdftitle={Genetic architecture of transcriptome regulation and orthogonal tissue decompositon},
            colorlinks=true,
            citecolor=blue,
            urlcolor=blue,
            linkcolor=magenta,
            pdfborder={0 0 0}}
\urlstyle{same}  % don't use monospace font for urls
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\setcounter{secnumdepth}{0}

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\newcommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}
  \title{Genetic architecture of transcriptome regulation and orthogonal tissue
decompositon}
  \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
  \author{Heather E. Wheeler\textsuperscript{1}, Nicholas
Knoblauch\textsuperscript{2}, GTEx Consortium, Nancy J.
Cox\textsuperscript{3}, Dan L. Nicolae\textsuperscript{1}, Hae Kyung
Im\textsuperscript{1}}
  \preauthor{\centering\large\emph}
  \postauthor{\par}
  \predate{\centering\large\emph}
  \postdate{\par}
  \date{2015-05-29 14:29:36 \textsuperscript{1}Department of Medicine,
University of Chicago, \textsuperscript{2}Committee on Genetics,
Genomics, and Systems Biology, University of Chicago,
\textsuperscript{3}Division of Genetic Medicine, Vanderbilt University}



\begin{document}

\maketitle


\section{Abstract}\label{abstract}

\emph{Lorem ipsum dolor sit amet, est ad doctus eligendi scriptorem. Mel
erat falli ut. Feugiat legendos adipisci vix at, usu at laoreet
argumentum suscipiantur. An eos adhuc aliquip scriptorem, te adhuc dolor
liberavisse sea. Ponderum vivendum te nec, id agam brute disputando
mei.}

\section{Introduction}\label{introduction}

cite Kruglyak review, Price PGen 11, Wright NatGen 14

\section{Results}\label{results}

\subsection{Local genetic variation explains a large proportion of gene
expression
variance}\label{local-genetic-variation-explains-a-large-proportion-of-gene-expression-variance}

We estimated the heritability of gene expression in whole blood from the
Depression Genes and Networks (DGN) cohort (n=922) {[}1{]} using a
mixed-effects model (see Methods) and calculated variances using
restricted maximum likelihood as implemented in GCTA {[}2{]}. We fit a
joint model with a local and a global genetic relationship matrix (GRM).
The local GRM was derived from SNPs within 1 Mb of each gene and the
global GRM was derived from SNPs that are located on non-gene
chromosomes and are eQTLs in the Framingham Heart Study (FHS) cohort
(n=5257, FDR \textless{} 0.05) {[}3{]}. The mean local
h\textsuperscript{2} was 0.130 and 54.6\% of genes had a positive 95\%
confidence interval (CI), while the mean global h\textsuperscript{2} was
0.076 and just 4.2\% of genes had a positive CI (Fig 1). The maximum
local h\textsuperscript{2} was 0.93 with a standard error (SE) of 0.009
while the maximum global h\textsuperscript{2} was 0.91 with a SE of
0.16. Similar results were observed for the 1194 genes with
\emph{trans}-eQTLs (FHS FDR \textless{} 0.05) when the global GRM was
limited to known \emph{trans}-eQTLs (Fig 2). That is, the mean local
h\textsuperscript{2} was 0.133 and 61.3\% of genes had a positive 95\%
confidence interval (CI), while the mean \emph{trans}
h\textsuperscript{2} was just 0.021 and 4.2\% of genes tested had a
positive CI.

\subsection{Cross-tissue and tissue-specific gene expression by
orthogonal tissue
decomposition}\label{cross-tissue-and-tissue-specific-gene-expression-by-orthogonal-tissue-decomposition}

In order to better understand the context specificity of gene expression
regulation, we developed a method called orthogonal tissue decomposition
(OTD), which uses a mixed effects model to generate cross-tissue and
tissue-specific gene expression levels (see Methods). Using a marginal
model with just the local GRM, we estimated the local
h\textsuperscript{2} of cross-tissue gene expression and tissue-specific
gene expression in the nine tissues with the most samples. The
cross-tissue heritabilities were larger and the standard errors were
smaller than the tissue-specific estimates (Fig 3). The percentage of
h\textsuperscript{2} estimates with positive CIs was much larger for
cross-tissue expression (17.3\%) than the tissue-specific expressions
(all less than 3\%, Fig 4).

We also compared the cross-tissue h\textsuperscript{2} from the OTD to
h\textsuperscript{2} estimates from the pre-OTD measures of gene
expression in each of the nine tissues, which we term tissue-wide
expression. Again, the cross-tissue heritabilities were larger and the
standard errors were smaller than the tissue-wide estimates (Fig 5),
though less striking than the tissue-specific comparison. The percentage
of tissue-wide h\textsuperscript{2} estimates with positive CIs ranged
from 4.4-8.6\% and thus were all larger than the tissue-specific postive
CI percentages, but smaller than the cross-tissue percentage (Fig 6).

\subsection{The effect of local genetic variation on gene expression is
sparse rather than
polygenic}\label{the-effect-of-local-genetic-variation-on-gene-expression-is-sparse-rather-than-polygenic}

We performed 10-fold cross-validation using the elastic net {[}4{]} to
test the predictive performance of local SNPs for gene expression across
a range of mixing parameters, \(\alpha\). The \(\alpha\) that gives the
largest cross-validation R\textsuperscript{2} informs the sparsity of
each gene expression trait. That is, at one extreme, if the optimal
\(\alpha=0\) (equivalent to ridge regression), the gene expression trait
is highly polygenic, whereas if the optimal \(\alpha=1\) (equivalent to
LASSO), the trait is highly sparse. We found that for most gene
expression traits, the cross-validated R\textsuperscript{2} was
suboptimal for \(\alpha=0\) and \(\alpha=0.05\), but nearly identically
optimal for \(\alpha=0.5\) and \(\alpha=1\) in the DGN cohort (Fig 7).
Therefore, the effect of local genetic variation on gene expression is
sparse rather than polygenic.

\section{Discussion}\label{discussion}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\itemsep1pt\parskip0pt\parsep0pt
\item
  local + trans heritability (others have done this)
\item
  trans heritability estimates not reliable, proportion not reliable
\item
  orthogonal tissue decomposition
\item
  cross tissue + tissue specific heritability -- estimates higher and se
  lower for cross-tissue The tissue availability is unbalanced because
  of the difficulties of sample collection and the uneven quality of the
  tissues. Furthermore, by using a mixed effects model to create
  cross-tissue expression, we borrow information across tissues, which
  should increase our power to detect associations and achieve better
  predictive models.
\item
  elastic net mixing parameter (alpha) as measure of
  polygenicity/sparsity
\end{enumerate}

Future

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\itemsep1pt\parskip0pt\parsep0pt
\item
  simulation to show sparsity well represented by alpha
\item
  use number of PC's (computed using only local snps) that maximize
  prediction performance. This will count independent signals.
\item
  FHS heritability. could this improve trans heritability?
\end{enumerate}

\section{Methods}\label{methods}

\subsection{Genomic and Transcriptomic
Data}\label{genomic-and-transcriptomic-data}

\subsubsection{DGN Dataset}\label{dgn-dataset}

We obtained whole blood RNA-Seq and genome-wide genotype data for 922
individuals from the Depression Genes and Networks (DGN) cohort {[}1{]},
all of European ancestry. For our analyses, we used the HCP (hidden
covariates with prior) normalized gene-level expression data used for
the \emph{trans}-eQTL analysis in Battle et al. {[}1{]} and downloaded
from the NIMH repository. The 922 individuals were unrelated (all
pairwise \(\hat{\pi}\) \textless{} 0.05) and thus all included in
downstream analyses. Imputation of approximately 650K input SNPs (minor
allele frequency {[}MAF{]} \textgreater{} 0.05, Hardy-Weinberg
Equilibrium {[}P \textgreater{} 0.05{]}, non-ambiguous strand {[}no A/T
or C/G SNPs{]}) was performed on the University of Michigan
Imputation-Server
(\url{https://imputationserver.sph.umich.edu/start.html}) {[}5,6{]} with
the following parameters: 1000G Phase 1 v3 ShapeIt2 (no singletons)
reference panel, SHAPEIT phasing, and EUR population. Approximately 1.9M
non-ambiguous strand SNPs with MAF \textgreater{} 0.05, imputation
R\textsuperscript{2} \textgreater{} 0.8 and, to reduce computational
burden, inclusion in HapMap Phase II were retained for subsequent
analyses.

\subsubsection{GTEx Dataset}\label{gtex-dataset}

We obtained RNA-Seq gene expression levels from 8555 tissue samples (53
unique tissue types) from 544 unique subjects in the GTEx Project
{[}7{]} data release on 2014-06-13. Of the individuals with gene
expression data, genome-wide genotypes (imputed with 1000 Genomes) were
available for 450 individuals. While all 8555 tissue samples were used
in the OTD model (described below) to generate cross-tissue and
tissue-specific components of gene expression, we used the nine tissues
with the largest sample sizes when quantifying tissue-specific effects.
Tissues and sample sizes (both RNA-seq and genotypes available) included
cross-tissue (n=450), skeletal muscle (n=361), whole blood (n=339), skin
from the sun-exposed portion of the lower leg (n=303), subcutaneous
adipose (n=298), tibial artery (n=285), lung (279), thyroid (n=279),
tibial nerve (n=256) and left ventricle heart (n=190). Approximately
2.6M non-ambiguous strand SNPs included in HapMap Phase II were retained
for subsequent analyses.

\subsection{Partitioning local and global heritability of gene
expression}\label{partitioning-local-and-global-heritability-of-gene-expression}

To investigate the proximity of gene expression regulation to each gene,
we partitioned the proportion of gene expression variance explained by
SNPs in the DGN cohort into two components: local (SNPs within 1Mb of
the gene) and global (eQTLs on non-gene chromosomes) as defined by the
GENCODE {[}8{]} version 12 gene annotation. We calculated the proportion
of the variance (narrow-sense heritability) explained by each component
using the following mixed-effects model:

\[ Y_g = \sum_{k = \in local}w_{k,g} X_k + \sum_{k = \in global}w_{k,g} X_k + \epsilon \]

Assuming a random effects for \(w_{k,g} \approx N(0, \sigma^2_w)\) and
\(\epsilon \approx N(0, \sigma^2_{\epsilon} I_n)\), where \(I_n\) is the
identity matrix, we calculated the total variability explained by local
and global components by estimating \(\sigma^2_w\) with restricted
maximum likelihood (REML) using GCTA software {[}2{]}. For heritability
analyses in the GTEx cohort, we removed the \(global\) term from the
model and only estimated marginal \(local\) h\textsuperscript{2} due to
the smaller sample sizes of both cross-tissue and tissue-specific
expression levels compared to DGN.

\subsection{Orthogonal tissue
decomposition}\label{orthogonal-tissue-decomposition}

To better understand the context specificity of gene expression
regulation, we developed a method called orthogonal tissue decomposition
(OTD). This approach is an extension of our method to develop an
intrinsic growth phenotype {[}9{]}. We applied OTD to GTEx Project
{[}7{]} data and decomposed the expression of each gene into
cross-tissue and tissue-specific components. The tissue availability is
unbalanced across individuals because of the difficulties of sample
collection and the uneven quality of the tissues. OTD decomposes the
expression traits into orthogonal components as represented by the
following model:

\[ Y_i = T_{i,cross} + T_{i,tissue} \]

Specifically, to generate cross-tissue and tissue-specific expression
levels, we used the \texttt{lmer} function in the R {[}10{]} package
\texttt{lme4} {[}11,12{]} to fit the following mixed-effects model:

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{fit <-}\StringTok{ }\NormalTok{lme4::}\KeywordTok{lmer}\NormalTok{(expression ~}\StringTok{ }\NormalTok{(}\DecValTok{1}\NormalTok{|SUBJID) +}\StringTok{ }\NormalTok{TISSUE +}\StringTok{ }\NormalTok{GENDER +}\StringTok{ }\NormalTok{PEERs)}
\end{Highlighting}
\end{Shaded}

The model included tissue-wide gene expression levels in 8555 GTEx
tissue samples from 544 unique subjects. A total of 17,647
Protein-coding genes (defined by GENCODE {[}8{]} version 18) with a mean
gene expression level across tissues greater than 0.1 RPKM (reads per
kilobase of transcript per million reads mapped) were included in the
model. \texttt{SUBJID} was a random effect and the covariates
\texttt{TISSUE}, \texttt{GENDER}, and \texttt{PEERs} were fixed effects
used to predict tissue-wide expression levels (\texttt{expression} in
the model). \texttt{PEERs} included the top 15 PEER factors estimated
across all tissues using the R package \texttt{PEER} {[}13{]} to control
for batch effects and experimental confounders. Cross-tissue expression
was defined as the random effects from the model (\texttt{ranef(fit)})
and tissue-specific expression as the residuals (\texttt{resid(fit})).

\subsection{Determining polygenicity versus sparsity using the elastic
net}\label{determining-polygenicity-versus-sparsity-using-the-elastic-net}

\textbf{WORK ON THIS} \texttt{glmnet} solves the following problem {\[
\min_{\beta_0,\beta} \frac{1}{N} \sum_{i=1}^{N} w_i l(y_i,\beta_0+\beta^T x_i) + \lambda\left[(1-\alpha)||\beta||_2^2/2 + \alpha ||\beta||_1\right],
\]} over a grid of values of {\(\lambda\)} covering the entire range.

The elastic-net penalty is controlled by {\(\alpha\)}, and bridges the
gap between lasso ({\(\alpha=1\)}, the default) and ridge
({\(\alpha=0\)}). The tuning parameter {\(\lambda\)} controls the
overall strength of the penalty.

We performed 10-fold cross-validation using the elastic net {[}4{]} to
test the predictive performance of local SNPs for gene expression across
a range of mixing parameters, \(\alpha\).

\subsection{Enrichment analysis}\label{enrichment-analysis}

\begin{itemize}
\itemsep1pt\parskip0pt\parsep0pt
\item
  For top CT and TS genes:
\end{itemize}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\itemsep1pt\parskip0pt\parsep0pt
\item
  GO enrichment
\item
  GWAS catalog enrichment (i.e.~top T2D, T1D, schizo, etc. genes)
\end{enumerate}

\subsection{Tables}\label{tables}

\begin{longtable}[c]{@{}lrrrr@{}}
\caption{This is a GLM summary table.}\tabularnewline
\toprule
& Estimate & Std. Error & t value &
Pr(\textgreater{}\textbar{}t\textbar{})\tabularnewline
\midrule
\endfirsthead
\toprule
& Estimate & Std. Error & t value &
Pr(\textgreater{}\textbar{}t\textbar{})\tabularnewline
\midrule
\endhead
(Intercept) & 0.09 & 0.11 & 0.90 & 0.37\tabularnewline
x & 1.94 & 0.10 & 19.06 & 0.00\tabularnewline
\bottomrule
\end{longtable}

\section{Figures}\label{figures}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/jointH2-1.pdf}
\caption{DGN whole blood expression joint heritability
(h\textsuperscript{2}). Local (SNPs within 1 Mb of each gene) and global
(SNPs that are eQTLs in the Framingham Heart Study on other chromosomes
{[}FDR \textless{} 0.05{]}) h\textsuperscript{2} for gene expression
were jointly estimated. (\textbf{A}) Global h\textsuperscript{2}
compared to local h\textsuperscript{2} per gene. (\textbf{B}) Local and
(\textbf{C}) global gene expression h\textsuperscript{2} estimates
ordered by increasing h\textsuperscript{2}. The 95\% confidence interval
(CI) of each h\textsuperscript{2} estimate is in gray and genes with a
lower bound greater than zero are in blue.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/transH2-1.pdf}
\caption{DGN whole blood expression joint heritability
(h\textsuperscript{2}) with known trans-eQTLs. Local (SNPs within 1 Mb
of each gene) and known trans (SNPs that are trans-eQTLs in the
Framingham Heart Study for each gene {[}FDR \textless{} 0.05{]})
h\textsuperscript{2} for gene expression were jointly estimated.
(\textbf{A}) Known trans h\textsuperscript{2} compared to local
h\textsuperscript{2} per gene. (\textbf{B}) Local and (\textbf{C}) known
trans gene expression h\textsuperscript{2} estimates ordered by
increasing h\textsuperscript{2}. The 95\% confidence interval (CI) of
each h\textsuperscript{2} estimate is in gray and genes with a lower
bound greater than zero are in blue.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/TSotdH2SE-1.pdf}
\caption{Cross-tissue and tissue-specific comparison of heritability
(h\textsuperscript{2}, \textbf{A}) and standard error (SE, \textbf{B})
estimation. Cross-tissue local h\textsuperscript{2} is estimated using
the cross-tissue component (random effects) of the mixed effects model
for gene expression and SNPs within 1 Mb of each gene. Tissue-specifc
local h\textsuperscript{2} is estimated using the tissue-specific
component (residuals) of the mixed effects model for gene expression for
each respective tissue and SNPs within 1 Mb of each gene.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/otdTSh2-1.pdf}
\caption{Cross-tissue heritability (h\textsuperscript{2}) compared to
tissue-specific h\textsuperscript{2}. Cross-tissue local
h\textsuperscript{2} is estimated using the cross-tissue component
(random effects) of the mixed effects model for gene expression and SNPs
within 1 Mb of each gene. Tissue-specifc local h\textsuperscript{2} is
estimated using the tissue-specific component (residuals) of the mixed
effects model for gene expression for each respective tissue and SNPs
within 1 Mb of each gene.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/TWotdH2SE-1.pdf}
\caption{Cross-tissue and tissue-wide comparison of heritability
(h\textsuperscript{2}, \textbf{A}) and standard error (SE, \textbf{B}).
Cross-tissue local h\textsuperscript{2} is estimated using the
cross-tissue component (random effects) of the mixed effects model for
gene expression and SNPs within 1 Mb of each gene. Tissue-wide local
h\textsuperscript{2} is estimated using the measured gene expression for
each respective tissue and SNPs within 1 Mb of each gene.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/otdTWh2-1.pdf}
\caption{Cross-tissue heritability (h\textsuperscript{2}) compared to
tissue-wide h\textsuperscript{2}. Cross-tissue local
h\textsuperscript{2} is estimated using the cross-tissue component
(random effects) of the mixed effects model for gene expression and SNPs
within 1 Mb of each gene. Tissue-wide local h\textsuperscript{2} is
estimated using the measured gene expression for each respective tissue
and SNPs within 1 Mb of each gene.}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics{GenArch_manuscript_files/figure-latex/EN-1.pdf}
\caption{Cross-validated predictive performance across the elastic net.
(\textbf{A}) 10-fold cross-validated R\textsuperscript{2} of predicted
vs.~observed expression in DGN whole blood compared to a range of
elastic net mixing parameters (\(\alpha\)) for genes on chromosome 22
with R\textsuperscript{2} \textgreater{} 0.3. (\textbf{B}) Predictive
R\textsuperscript{2} difference between LASSO (\(\alpha = 1\)) and
several values of \(\alpha\) compared to LASSO predictive
R\textsuperscript{2} for 341 genes on chromosome 22.}
\end{figure}

\section{Supplemental Figures}\label{supplemental-figures}

\section*{References}\label{references}
\addcontentsline{toc}{section}{References}

1. Battle A, Mostafavi S, Zhu X, Potash JB, Weissman MM, McCormick C, et
al. Characterizing the genetic basis of transcriptome diversity through
RNA-sequencing of 922 individuals. Genome Research. Cold Spring Harbor
Laboratory Press; 2013;24: 14--24.
doi:\href{http://dx.doi.org/10.1101/gr.155192.113}{10.1101/gr.155192.113}

2. Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A tool for genome-wide
complex trait analysis. The American Journal of Human Genetics. Elsevier
BV; 2011;88: 76--82.
doi:\href{http://dx.doi.org/10.1016/j.ajhg.2010.11.011}{10.1016/j.ajhg.2010.11.011}

3. Zhang X, Joehanes R, Chen BH, Huan T, Ying S, Munson PJ, et al.
Identification of common genetic variants controlling transcript isoform
variation in human whole blood. Nat Genet. Nature Publishing Group;
2015;47: 345--352.
doi:\href{http://dx.doi.org/10.1038/ng.3220}{10.1038/ng.3220}

4. Zou H, Hastie T. Regularization and variable selection via the
elastic net. Journal of the Royal Statistical Society: Series B
(Statistical Methodology). Wiley-Blackwell; 2005;67: 301--320.
doi:\href{http://dx.doi.org/10.1111/j.1467-9868.2005.00503.x}{10.1111/j.1467-9868.2005.00503.x}

5. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and
accurate genotype imputation in genome-wide association studies through
pre-phasing. Nat Genet. Nature Publishing Group; 2012;44: 955--959.
doi:\href{http://dx.doi.org/10.1038/ng.2354}{10.1038/ng.2354}

6. Fuchsberger C, Abecasis GR, Hinds DA. Minimac2: Faster genotype
imputation. Bioinformatics. Oxford University Press (OUP); 2014;31:
782--784.
doi:\href{http://dx.doi.org/10.1093/bioinformatics/btu704}{10.1093/bioinformatics/btu704}

7. Ardlie KG, Deluca DS, Segre AV, Sullivan TJ, Young TR, Gelfand ET, et
al. The genotype-tissue expression (GTEx) pilot analysis: Multitissue
gene regulation in humans. Science. American Association for the
Advancement of Science (AAAS); 2015;348: 648--660.
doi:\href{http://dx.doi.org/10.1126/science.1262110}{10.1126/science.1262110}

8. Harrow J, Frankish A, Gonzalez JM, Tapanari E, Diekhans M, Kokocinski
F, et al. GENCODE: The reference human genome annotation for the ENCODE
project. Genome Research. Cold Spring Harbor Laboratory Press; 2012;22:
1760--1774.
doi:\href{http://dx.doi.org/10.1101/gr.135350.111}{10.1101/gr.135350.111}

9. Im HK, Gamazon ER, Stark AL, Huang RS, Cox NJ, Dolan ME. Mixed
effects modeling of proliferation rates in cell-based models:
Consequence for pharmacogenomics and cancer. Akey JM, editor. PLoS
Genetics. Public Library of Science (PLoS); 2012;8: e1002525.
doi:\href{http://dx.doi.org/10.1371/journal.pgen.1002525}{10.1371/journal.pgen.1002525}

10. R Core Team. R: A language and environment for statistical computing
{[}Internet{]}. Vienna, Austria: R Foundation for Statistical Computing;
2015. Available: \url{http://www.R-project.org/}

11. Bates D, Maechler M, Bolker B, Walker S. lme4: Linear mixed-effects
models using eigen and s4 {[}Internet{]}. 2014. Available:
\url{http://CRAN.R-project.org/package=lme4}

12. Bates D, Maechler M, Bolker BM, Walker S. lme4: Linear mixed-effects
models using eigen and s4 {[}Internet{]}. 2014. Available:
\url{http://arxiv.org/abs/1406.5823}

13. Stegle O, Parts L, Piipari M, Winn J, Durbin R. Using probabilistic
estimation of expression residuals (PEER) to obtain increased power and
interpretability of gene expression analyses. Nature Protocols. Nature
Publishing Group; 2012;7: 500--507.
doi:\href{http://dx.doi.org/10.1038/nprot.2011.457}{10.1038/nprot.2011.457}

\end{document}
